build/db_schemas/scd/upto-v3.2.0-add_ovn_columns.sql

```diff
@@ -0,0 +1,14 @@
+ALTER TABLE scd_operations
+    ADD COLUMN IF NOT EXISTS ovn       STRING
+        CHECK (ovn != ''), -- ovn must be NULL if unspecified, not an empty string
+    ADD COLUMN IF NOT EXISTS past_ovns STRING[] NOT NULL
+        DEFAULT ARRAY []::STRING[]
+        CHECK (
+            array_position(past_ovns, NULL) IS NULL AND
+            array_position(past_ovns, '') IS NULL AND
+            array_position(past_ovns, ovn) IS NULL
+            ); -- past_ovns must not contain NULL elements, empty strings or current ovn
```

Comment:
Side thought: how big do we expect these arrays to grow? 

I don't believe that enforcing the constraint will ever become a problem, but this still makes me wonder how many modifications of an OIR (which is what will cause this array to grow) we should expect in the worst case.

Reply:
I do not know. But given that we don't use it to query anything, I don't expect its size to have any significant impact.